samples/excel/89-preview-apis/events-worksheet-activated.yaml

```diff
@@ -19,7 +19,8 @@ script:
 
                 await context.sync();
 
-                OfficeHelpers.UI.notify("A handler has been registered for the OnActivate event.");   
+                OfficeHelpers.UI.notify("A handler has been registered for the OnActivate event",
+                    "Try changing a cell value in the table, and watch the console output.");   
```

Comment:
The second part of this error looks like a copy/paste error. i.e., it seems to describe the onChanged event, not the onActivate event. 

The second text string should instead say something like:   "Try selecting a different sheet, and watch the console output."

Reply:
Also, suggest that you add this second text string the UI.notify message in the **registerOnDeactivateHandler()** function as well.

Reply:
Finally -- if your intention in updating this UI.notify message here was to address my feedback on your previous PR ([#119](https://github.com/OfficeDev/office-js-snippets/pull/119)), that feedback was for the "Register onAdded event handler" (which is in the "Events - Worksheet changed" snippet).

"....as we discussed, whenever you have time: please update the UI notification for the "Register onAdded event handler" button click"

Reply:
Kim – this is a merge from master to prod.   All the code snippets have been code reviewed.   Let’s not mix a fork review with merge from master to prod.  I can make changes next time I do a PR to merge my fork to master.

Reply:
@kbrandl -- change is in pull 122 now